scripts/accessibility/helpers.R

```diff
@@ -18,7 +18,7 @@ add_inc_group <- function(df) {
   df0 <- df %>%
     filter(!age_group %in% "age_7-17") %>%
     arrange(income) %>%
-    mutate(income_group = ceiling(10 * row_number() / n()))
+    mutate(income_group = ceiling(5 * row_number() / n()))
```

Comment:
Consider making this `ceiling` line a function. Right now, it is difficult to tell what it does. Also, would `quantile` work here?

Reply:
Seems that `dplyr` has readily available `ntile` function, which should do what I was looking for.  I always get surprised with the extent of these libraries. :)

Reply:
That is new for me too! :)